reference/conanfile/methods.rst

```diff
@@ -720,6 +724,18 @@ instance of the package manager. It validates whether your current settings are
 cross-building, and if so, it will update the package name to be installed according to
 ``self.settings.arch``.
 
+To install more than one package at once:
+
+..  code-block:: python
+
+            packages = [("vim", "nano", "emacs"), "firefox", "chromium"]
+            installer = SystemPackageTool()
+            installer.install_packages(packages)
+            # e.g. apt-get install -y --no-recommends vim firefox chromium
+
+The ``install_packages`` will install one of text editors (first available), because it's a nested tuple, and it will install both web browsers.
```

Comment:
```suggestion
The ``install_packages`` will install the first text editor available (only one) following the tupple order, while it will install both web browsers.
```

Reply:
Done